course_catalog/constants.py

```diff
@@ -31,6 +31,7 @@ class PlatformType(Enum):
     bootcamps = "bootcamps"
     xpro = "xpro"
     oll = "oll"
+    youtube = "youtube"
```

Comment:
Add `youtube = "YouTube"` to `class OfferedBy(Enum)` as well?

Reply:
These videos will actually be offered by existing values (e.g. OCW, MITx)